lib/control-app.js

```diff
@@ -98,10 +100,19 @@ module.exports = function (config, cb) {
     })
   }
 
+  // support IE9+ with runtime feature detection
+  var oldBuffer = require.resolve('buffer/')
+  var newBuffer = browserifyBuiltins.buffer
+  var chooseImplementation = path.join(__dirname, '../client/buffer.js')
+  bundler.require(oldBuffer, { expose: 'buffer@4' })
+  bundler.require(newBuffer, { expose: 'buffer@5' })
+  bundler.require(chooseImplementation, { expose: 'buffer' })
```

Comment:
Not sure this complexity is worth it.

Just a thought, we could have IE9 and 10 request something like `/airtap/client.js?ie=1` and serve them a different bundle?

Reply:
Yeah, at this point just unconditionally using `buffer@4` would probably be a smaller maintenance burden, though it might become a problem already if we ever update tap-parser. I like the idea of just serving different bundles!

Reply:
(Or a yaml/cli option to use the old `buffer` would be easier to implement, but may suggest that The Project will invest in supporting IE9, which may not be the case.)

Reply:
> may suggest that The Project will invest in supporting IE9

That's fine and already true (whether it's by maintaining 1x, or by the alias approach, the `?ie=1` approach, or the yaml/cli option approach you just suggested).

If it's not too much trouble, could you open a separate PR that takes the option approach? So that we can evaluate which is the cleanest.

Reply:
If we want to officially support IE9, the easiest approach would be to unconditionally use `buffer@4` in the client code. Picking the implementation at runtime was intended so the client code could evolve without caring about IE9/10. But if we want to commit to supporting them, the client code needs to work with `buffer@4` anyway, so switching between both implementations has no benefit. I would like that, maybe I was too cautious by assuming that you wouldn't want to care about very old browsers :)

Reply:
#274